virt/virt-getting-started.adoc

```diff
@@ -0,0 +1,80 @@
+
+[discrete]
+[id="additional-resources-planning-and-installing"]
+[role="_additional-resources"]
+=== Planning and installation resources
+* xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider].
+* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage for virtual machines].
+* xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#installing-the-kubernetes-nmstate-operator-cli[Installing the Kubernetes NMState Operator].
+* xref:../virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc#virt-specifying-nodes-for-vms[Specifying nodes for virtual machines].
+* xref:../virt/virt-using-the-cli-tools.adoc#virt-virtctl-commands_virt-using-the-cli-tools[`Virtctl` commands].
+
+[id='configuring-networks_{context}']
+=== Configuring secondary networks
+
+Configuring a secondary network is optional. VMs are connected to the default pod network.
+
+Configure a Linux bridge network:
+
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-linux-bridge-nncp_virt-attaching-vm-multiple-networks[Create a node network configuration policy].
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-linux-bridge-nad-web_virt-attaching-vm-multiple-networks[Create a network attachment definition for the Linux bridge network].
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-vm-creating-nic-web_virt-attaching-vm-multiple-networks[Attach a VM to the Linux bridge network].
+
+Configure an SR-IOV network:
+
+* xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[Install the SR-IOV Operator].
+* xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[Configure an SR-IOV network device].
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[Attach a VM to the SR-IOV network].
+
+[id="creating-and-managing-vms_{context}"]
+== Creating and managing virtual machines
+
+Create virtual machines (VMs) by using the web console:
+
+* xref:../virt/virtual_machines/virt-create-vms.adoc#virt-quick-creating-vm_virt-create-vms[Quick create a VM].
+* xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-custom-template_virt-create-vms[Customize a template to create a VM].
+
+Connect to the VMs:
+
+* Connect to the xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[serial console] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-vnc-console_virt-accessing-vm-consoles[VNC console] of a VM by using the web console.
```

Comment:
The landing page that this link 'serial console' navigates to is the same as the one for 'VNC console', 'Connect to a VM by using SSH' and 'Connect to a Windows VM by using RDP' . If you have a link then it should navigate to the specific text. Seems redundant for it to navigate to another page with another link.

Reply:
Same issue. The linked header is correct, but it is barely visible at the top of the screen. This is a tooling issue, not a docs issue. I might try submitting a bug to the CCSOPS team, but it would not be high priority.